src/app/code/community/Shopgate/Cloudapi/Model/Frontend/Observer/Layout.php

```diff
@@ -21,6 +21,16 @@
  */
 class Shopgate_Cloudapi_Model_Frontend_Observer_Layout
 {
+    /**
+     * Define custom handles
+     */
+    const CUSTOM_HANDLES = array(
+        array(
+            'path'   => '/customer/account/login',
+            'handle' => 'shopgate_cloudapi_customer_account_login'
+        )
+    );
+
     /**
      * @param Varien_Event_Observer $observer
      * @return $this
```

Comment:
ugh, in the previous I forgot to mention, should have a space between param and return. Should also declare the `Varien_Exception` throws

Reply:
changed